class/keycloak.yml

```diff
@@ -9,3 +14,57 @@ parameters:
           - keycloak/component/main.jsonnet
         input_type: jsonnet
         output_path: keycloak/
+      - output_path: keycloak/01_keycloak_helmchart
+        input_type: helm
+        output_type: yaml
+        input_paths:
+          - keycloak/helmcharts/keycloak
+        helm_values:
+          statefulsetLabels: ${keycloak:labels}
+          resources: ${keycloak:resources}
+          # extraEnv *MUST* be a string, as it's fed through a templating
+          # function.
+          extraEnv: |
+            - name: JAVA_OPTS
+              value: >-
+                        -XX:+UseContainerSupport
+                        -XX:MaxRAMPercentage=50.0
+                        -Djava.net.preferIPv4Stack=true
+                        -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS
+                        -Djava.awt.headless=true
+                        ${keycloak:extraJavaOpts}
+            - name: KEYCLOAK_STATISTICS
+              value: ${keycloak:monitoring:statistics}
+          extraEnvFrom: |
+            - secretRef:
+                name: ${keycloak:admin:secretname}
+            - secretRef:
+                name: $[ self:keycloak:postgres:external:secretname if exports:keycloak:postgres:builtin == false ]
```

Comment:
Unfortunately I don't think this will work as expected. The inventory queries return `$node-name: key value` dicts, and not just the referenced value.

Reply:
yep, I thought I had tested this before my vacation, but this doesn't appear to work as I wanted it to. I'm switching to always creating the secret, and just not adding any key-value pairs when `keycloak:postgres:builtin` is `true`.

Reply:
The secret is created always, but is empty, when`keycloak:postgres:builtin` is `true`.